lms/djangoapps/support/static/support/js/collections/certificate.js

```diff
@@ -7,14 +7,23 @@
 
                 initialize: function(options) {
                     this.userQuery = options.userQuery || '';
+                    this.courseID = options.courseID || '';
                 },
 
                 setUserQuery: function(userQuery) {
                     this.userQuery = userQuery;
                 },
 
+                setCourseQuery: function(courseQuery) {
+                    this.courseID = courseQuery;
```

Comment:
No, It is not related to Backbone convention. In old implementation, It was mentioned as userQuery so I just continued with that convention in case of course. However I will udpate it as suggested.